ui/src/config/section/storage.js

```diff
@@ -443,54 +443,37 @@ export default {
       ]
     },
     {
-      name: 'backup',
```

Comment:
We need backup also right! may be by mistake. Can you keep this and add new sections for buckets